.github/workflows/build_and_test.yml

```diff
@@ -5311,6 +5311,17 @@ jobs:
         cd OpenDDS
         call setenv.cmd
         perl tools\scripts\show_build_config.pl
+    - name: setup gtest
+      shell: cmd
+      run: |
+        cd OpenDDS/tests/googletest
+        git submodule init
+        git submodule update
+        mkdir build
+        cd build
+        mkdir install
+        cmake -DCMAKE_INSTALL_PREFIX=./install -DBUILD_SHARED_LIBS=ON ..
```

Comment:
I think these flags (and the flags for the other broken-apart "test" jobs) need to match (one way or the other) the flags used in the build step above which currently don't use BUILD_SHARED_LIBS=ON